fast_forward/bonded_file_parser.py

```diff
@@ -0,0 +1,125 @@
+
+import re
+
+def read_interactions(lines):
+    interactions = {}
```

Comment:
@csbrasnett this reads a pycgtool like file right? That would become redundant with the ff template right?